.github/workflows/gbs_x64.yml

```diff
@@ -22,3 +22,7 @@ jobs:
       run: cp .github/workflows/tizen.gbs.conf ~/.gbs.conf
     - name: run GBS
       run: gbs build
+    - name: get nntrainer
+      run: git clone https://github.com/nnstreamer/nntrainer.git
```

Comment:
How about using checkout action instead of run git clone?
https://github.com/actions/checkout#checkout-multiple-repos-side-by-side

Reply:
Thanks, I updated it as you recommended.